tomcat/src/test/java/nextstep/org/apache/coyote/http11/Http11ProcessorTest.java

```diff
@@ -79,10 +80,88 @@ void login() throws IOException {
         final URL resource = getClass().getClassLoader().getResource("static/login.html");
         var expected = "HTTP/1.1 200 OK \r\n" +
                 "Content-Type: text/html;charset=utf-8 \r\n" +
-                "Content-Length: 3796 \r\n" +
+                "Content-Length: 3447 \r\n" +
                 "\r\n" +
                 new String(Files.readAllBytes(new File(resource.getFile()).toPath()));
 
         assertThat(socket.output()).isEqualTo(expected);
     }
+
+    @Test
+    void login() {
+        // given
+        final String httpRequest = String.join("\r\n",
+                "POST /login HTTP/1.1 ",
+                "Host: localhost:8080 ",
+                "Content-Length: 30",
+                "Connection: keep-alive ",
+                "",
+                "account=gugu&password=password");
+
+        final var socket = new StubSocket(httpRequest);
+        final Http11Processor processor = new Http11Processor(socket);
+
+        // when
+        processor.process(socket);
+
+        // then
+        final String output = socket.output();
+        assertAll(
+                () -> assertThat(output.contains("HTTP/1.1 302 Found \r\n")).isTrue(),
+                () -> assertThat(output.contains("Location: /index.html \r\n")).isTrue(),
+                () -> assertThat(output.contains("Set-Cookie: ")).isTrue()
+        );
```

Comment:
```suggestion
        assertThat(output).contains(
            "HTTP/1.1 302 Found \r\n",
            "Location: /index.html \r\n",
            "Set-Cookie: "
        );
```
위와 같이 검증문을 작성해도 됩니다.

Reply:
반영하였습니다!!